cli/run.go

```diff
@@ -239,6 +240,8 @@ func (r WorkflowRunner) RunWorkflowsWithSetupAndCheckForUpdate() (int, error) {
 func (r WorkflowRunner) runWorkflows(tracker analytics.Tracker) (models.BuildRunResultsModel, error) {
 	startTime := time.Now()
 
+	envfile.LogLargeEnvWarning()
```

Comment:
I guess this function is only logging a warning if the env list is too large, the current function name doesn't imply this, what about `LogLargeEnvWarningIfNeed` or something similar?